cmd/access/node_builder/access_node_builder.go

```diff
@@ -1271,6 +1273,10 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 			"program-cache-size",
 			defaultConfig.programCacheSize,
 			"[experimental] number of blocks to cache for cadence programs. use 0 to disable cache. default: 0. Note: this is an experimental feature and may cause nodes to become unstable under certain workloads. Use with caution.")
+		flags.BoolVar(&builder.checkPayerBalance,
```

Comment:
can you add a check in `ValidateFlags` to make sure that `builder.executionDataIndexingEnabled` is also set